src/machine/machine_rp2040_usb.go

```diff
@@ -77,7 +77,7 @@ func handleUSBIRQ(intr interrupt.Interrupt) {
 
 		if !ok {
 			// Stall endpoint?
-			sendStallViaEPIn(0)
+			SetStallEPIn(0)
```

Comment:
I know this is not directly related to the goal of this change, but it seems to me there's quite a lot of duplication between the `machine_rpxxxx_usb.go` files. Perhaps a `machine_rp2_usb.go` would make it easier to do changes now and in future.

Reply:
That is a really good point. I'm going to try putting together a PR to see how much can be consolidated